README.md

```diff
@@ -103,6 +103,18 @@ const isDangerous = hasTrojanSource({
 
 `hasTrojanSource` returns a boolean.
 
+# Use as a pre-commit hook
+
+To add this tool to your project as a [`pre-commit`](https://pre-commit.com) hook, try this sample configuration in `.pre-commit-config.yaml`:
+
+```yaml
+repos:
+  - repo: https://github.com/lirantal/anti-trojan-source
+    rev: v1.3.3  # choose the release you want
```

Comment:
@rpdelaney I assume you'll need to bump this one to the new version once this PR is merged or will it work still with 1.3.3?

Reply:
pre-commit will try to download whatever release the user enters here. You can update the readme with the latest version after every release if you like (or care to automate it), but if someone copy-pastes it should work (they'll just get an older version).

Reply:
Gotcha! Thanks for explaining :)